### PR TITLE
Read.md update for the user's guide

### PR DESCRIPTION
**Pull request by IIHermanSinghII**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +63,10 @@ development sessions, it is enough to issue
 from the project's top directory to enter the development environment.
 
 ## Running on Sigma Supercomputer
+When multiple materails need to be studied, its recommended to run it on a supercomputer. This software is able to run on NSC:s supercomputer Sigma in Linköping, Sweden. The following link explains how to get started with Sigma:
```
Comment (joast258) [inline README.md:66]: materials*

File: README.md [hunk 2]
```diff
@@ -108,3 +112,157 @@ then run
...
+
+potential
+
+	Arg:    Mandatory
+	Type:   String or list of strings
+	Desc:   Type of potential to be used on the materials.
+kim-model
+
+	Arg:    Mandatory if potential is openkim, otherwise optional.
+	Type:   String
```
Comment (joast258) [inline README.md:173]: "String or list of strings"

File: README.md [hunk 3]
```diff
@@ -108,3 +112,157 @@ then run
...
+
+	Arg:    Mandatory
+	Type:   String
+	Desc:   The selected path of the workspace. Note: if no example_workspace exist, a 
+	example_workspace will be created.
+
+ensemble
+
+	Arg:    Mandatory
+	Type:   String or list
```
Comment (joast258) [inline README.md:161]: "String or list of strings"

File: README.md [hunk 4]
```diff
@@ -108,3 +112,157 @@ then run
...
+# How to fill in the JSON file
+An example with documentation to the JSON file can be found in ...\Salsa-Dancing-Molecules\example_simulation\example.conf. A description of each argument is given below:
+
+material
+
+	 Arg:    Mandatory
+	 Type:   List or string
+	 Desc:   Materials to be simulated. If using custom materials, the names shall 
+	 correspond to atoms object(s) in the python file(s). If using materials project,use the 
+	 prefix "mp_" followed by the formula to be searched for.
```
Comment (joast258) [inline README.md:141]: Add that "mp_*element*" will grab every type of that element, while writing "mp_mp-*id*" will only grab that specific id.

**Review by joast258 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (IIHermanSinghII) [reply README.md:66]: done

File: README.md (hunk 2 above)
Comment (IIHermanSinghII) [reply README.md:173]: done

File: README.md (hunk 3 above)
Comment (IIHermanSinghII) [reply README.md:161]: done

File: README.md (hunk 4 above)
Comment (IIHermanSinghII) [reply README.md:141]: done

**Review by joast258 — APPROVED** [review]
LGTM

**Pull request merged**